kgreed/SBD.GeneralLedger
# Topic/use cache warmup

Fix #2:
This should finish up the cache warmup.

To test:
- Set Configuration to Debug or Release
- Set Platform to x64

Build and run the application via SBD.GL.Package
It now should use the cached assemblies

Gotcha: I've no idea why x86 is not working, but that has something to do with Sqlite. But that would be worth further investigation, cause x86 is about 30% faster in my tests.

Cause your project uses native dependencies, this uses an OutOfProcess mode (back to CLI) for cache generation.

This also introduces a generic VisualStudio .gitignore
